code/modules/chemistry/Chemistry-Tools.dm

```diff
@@ -415,6 +415,7 @@
 		qdel(src)
 
 	on_spin_emote(var/mob/living/carbon/human/user as mob)
+		. = ..()
```

Comment:
I just noticed that I did not pass the user parameter here and for drinking glasses. I think it still worked for me when I tested it, though?
Does byond automagically fill in the parameter for the parent call?

Reply:
Yes, it does.

Reply:
automagically passing args is generally better, as it provides more support for the future